__tests__/__snapshots__/flow.ts.snap

```diff
@@ -0,0 +1,77 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`it detects errors 1`] = `
+"Error ------------------------------------------------------------------------- __fixtures__/flow-errors-fixture.js:4:32
+
+Cannot assign object literal to \`css\` because:
+ - property \`unknownProperty\` is missing in \`Properties\` [1] but exists in object literal [2].
+ - in property \`flexGrow\`:
+    - Either string [3] is incompatible with enum [4].
+    - Or string [3] is incompatible with number [5].
+
+   __fixtures__/flow-errors-fixture.js:4:32
+                                        v
+      4| const css: CSS.Properties<*> = {
+      5|   flexGrow: 'invalid',
+      6|   unknownProperty: 'here',
+      7| };
+         ^ [2]
+
+References:
+   __fixtures__/flow-errors-fixture.js:4:12
+      4| const css: CSS.Properties<*> = {
+                    ^^^^^^^^^^^^^^^^^ [1]
+   __fixtures__/flow-errors-fixture.js:5:13
+      5|   flexGrow: 'invalid',
+                     ^^^^^^^^^ [3]
+   index.js.flow:4529:22
+   4529| type GlobalsNumber = Globals | number;
```

Comment:
These tests will fail every time these rows have changed. It needs to be addressed somehow.

Reply:
Maybe it's better to request a JSON output using the `--json` flag and do tests against parts of the result of that instead?

Reply:
I think it's fine to have them fail when the rows are changed. Updating the snapshot is easy; having it in diff reviews serves as a sanity test for future changes.